storage-iops-latency-throughput-demo/sample-managed-disks/managedDisksDemo.json

```diff
@@ -0,0 +1,279 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmAdminUserName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The administrative user on the Windows VM."
+      }
+    },
+    "vmAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The administrative user password."
+      }
+    },
+    "vNetName": {
+      "type": "string",
+      "defaultValue": "demo-vnet",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the virtual network resource."
+      }
+    },
+    "vmPIPDnsName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The DNS prefix assigned to the public IP address resource."
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/storage-iops-latency-throughput-demo/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "vNetPrefix": "10.0.0.0/16",
+    "vNetSubnet1Name": "subnet-01",
+    "vNetSubnet1Prefix": "10.0.0.0/24",
+    "diskName": [ "[concat(variables('vmName'),'-standard','-01')]", "[concat(variables('vmName'),'-standard','-02')]", "[concat(variables('vmName'),'-standard','-03')]", "[concat(variables('vmName'),'-premium','-01')]" ],
+    "diskCreateOption": "empty",
+    "diskAccountType": [ "Standard_LRS", "Standard_LRS", "Standard_LRS", "Premium_LRS" ],
+    "diskSizeGB": [ "100", "100", "100", "1023" ],
+    "vmImagePublisher": "MicrosoftWindowsServer",
+    "vmImageOffer": "WindowsServer",
+    "vmOSDiskName": "[concat(variables('vmName'),'-','OSDisk')]",
+    "vmSize": "Standard_DS3_v2",
+    "vmName": "demo-01",
+    "vmWindowsOSVersion": "2016-Datacenter",
+    "vmSubnetRef": "[concat(resourceId('Microsoft.Network/virtualNetworks', parameters('vNetName')), '/subnets/', variables('vNetSubnet1Name'))]",
```

Comment:
Had to add in a new vNetId variable first using resourceId(), and then the concat() function for the vmSubnetRef variable. I've found that trying to use resourceId() for a subnet throws an error - you need to concat() it with the resource ID of the vNet first